.env.example

```
@@ -1,41 +1,14 @@
+# Copy with: bun run env:setup
 
-# -----------------------------
-# Supabase
-# -----------------------------
-NEXT_PUBLIC_SUPABASE_URL=[https://your-project.supabase.co](https://your-project.supabase.co)
-NEXT_PUBLIC_SUPABASE_ANON_KEY=your-anon-key
-SUPABASE_SERVICE_ROLE_KEY=your-service-role-key  # Server-only!
+# Public browser configuration. Use a dedicated development Supabase project,
+# or replace both values with those from a complete local Supabase stack.
+NEXT_PUBLIC_SUPABASE_URL=https://replace-with-project-ref.supabase.co
+NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=sb_publishable_replace_with_your_project_key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)